### PR TITLE
[Keyboard] Add Dy16 with via compatibility

### PR DESCRIPTION
**Pull request by DreamyState**
Add keyboard Dreamystate Dy16 keyboard

## Description
Add keyboard Dy16;
Add compatibility for VIA on Dy 16.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dreamystate/dy16/config.h [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4459  // "DY" 0x4459
+#define PRODUCT_ID      0x0106  // "16" 0x0106
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Dreamystate Keyboard
+#define PRODUCT         Dreamystate Dy16 Keyboard  // show in Bluetooth and other devices management menu
+#define DESCRIPTION     Dy16
```
Comment (zvecr) [inline keyboards/dreamystate/dy16/config.h:27]:
```suggestion
```

File: keyboards/dreamystate/dy16/dy16.c [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	return process_record_user(keycode, record);
+}
+
+void led_set_kb(uint8_t usb_led) {
+	// put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+
+	led_set_user(usb_led);
+}
+
+
```
Comment (zvecr) [inline keyboards/dreamystate/dy16/dy16.c:45]:
```suggestion
```

File: keyboards/dreamystate/dy16/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  ),
+  
+  [10] = LAYOUT(
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS
+  )
+  */
```
Comment (zvecr) [inline keyboards/dreamystate/dy16/keymaps/default/keymap.c:105]:
```suggestion
  )
```

File: keyboards/dreamystate/dy16/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  ),
+  
+  [10] = LAYOUT(
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS
+  )
+  */
```
Comment (zvecr) [inline keyboards/dreamystate/dy16/keymaps/via/keymap.c:105]:
```suggestion
```

File: keyboards/dreamystate/dy16/keymaps/via/rules.mk [hunk 5]
```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+
+
```
Comment (zvecr) [inline keyboards/dreamystate/dy16/keymaps/via/rules.mk:4]:
```suggestion
```

File: keyboards/dreamystate/dy16/readme.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+	qmk compile -kb dreamystate/dy16 -km default
+	or
+	make dreamystate/dy16:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) 
+and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. 
+Brand new to QMK? 
+Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+
```
Comment (zvecr) [inline keyboards/dreamystate/dy16/readme.md:22]:
```suggestion
# Dreamystate Dy16 Keyboard

![Dy16Photo](https://i.imgur.com/jpnMay9.jpeg)
More: [Imgur](https://imgur.com/gallery/6tu9KdA).

A 16 key macropad, with USB C, RGB light and Buzzer、 Encoder in the future.

* Keyboard Maintainer: Dreamystate
* Hardware Supported: Dy16 PCB 

Make example for this keyboard (after setting up your build environment):

    make dreamystate/dy16:default

Flashing example for this keyboard:

    make dreamystate/dy16:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/dreamystate/dy16/rules.mk [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
+
+
+
```
Comment (zvecr) [inline keyboards/dreamystate/dy16/rules.mk:36]:
```suggestion
# MCU name
MCU = atmega32u4

# Bootloader selection
BOOTLOADER = atmel-dfu

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/dreamystate/dy16/info.json [hunk 8]
```diff
@@ -0,0 +1,36 @@
+{
+    "keyboard_name": "Dy16",
+    "url": "",
+    "maintainer": "qmk",
```
Comment (zvecr) [inline keyboards/dreamystate/dy16/info.json:4]:
```suggestion
    "maintainer": "Dreamystate",
```

File: keyboards/dreamystate/dy16/keymaps/default/keymap.c (hunk 3 above)
Comment (DreamyState) [reply keyboards/dreamystate/dy16/keymaps/default/keymap.c:105]:
```suggestion
  ),
  [1] = LAYOUT(
    KC_TRNS, KC_TRNS  ,  RESET,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS
  ),
  [2] = LAYOUT(
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS
  ),
 [3] = LAYOUT(
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS
  )

```

File: keyboards/dreamystate/dy16/config.h [hunk 9]
```diff
@@ -0,0 +1,236 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4459  // "DY" 0x4459
+#define PRODUCT_ID      0x0106  // "16" 0x0106
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Dreamystate Keyboard
+#define PRODUCT         Dreamystate Dy16 Keyboard  // show in Bluetooth and other devices management menu
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/config.h:26]:
```suggestion
#define PRODUCT         Dy16
```

File: keyboards/dreamystate/dy16/config.h [hunk 10]
```diff
@@ -0,0 +1,236 @@
...
+#define LCD_DATA3_PIN    7            //< pin for 4bit data bit 3
+#define LCD_RS_PORT      LCD_PORT     //< port for RS line
+#define LCD_RS_PIN       3            //< pin  for RS line
+#define LCD_RW_PORT      LCD_PORT     //< port for RW line
+#define LCD_RW_PIN       2            //< pin  for RW line
+#define LCD_E_PORT       LCD_PORT     //< port for Enable line
+#define LCD_E_PIN        1            //< pin  for Enable line
+#endif
+*/
+
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/config.h:232]:
```suggestion
```

File: keyboards/dreamystate/dy16/info.json [hunk 11]
```diff
@@ -0,0 +1,35 @@
+{
+    "keyboard_name": "Dy16",
+    "url": "",
+    "maintainer": "Dreamystate",
+    "width": 4,
+    "height": 5,
+    "layouts": {
+      "LAYOUT": {
+        "key_count": 20,
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/info.json:9]:
```suggestion
```

File: keyboards/dreamystate/dy16/keymaps/via/rules.mk (hunk 5 above)
Comment (fauxpark) [inline keyboards/dreamystate/dy16/keymaps/via/rules.mk:4]:
```suggestion
```

File: keyboards/dreamystate/dy16/keymaps/via/readme.md [hunk 12]
```diff
@@ -0,0 +1,3 @@
+# Via keymap for Dreamystate Dy16 Keyboard
+
+
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/keymaps/via/readme.md:3]:
```suggestion
```

File: keyboards/dreamystate/dy16/keymaps/default/readme.md [hunk 13]
```diff
@@ -0,0 +1,3 @@
+# Default keymap for Dreamystate Dy16 Keyboard
+
+
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/keymaps/default/readme.md:3]:
```suggestion
```

File: keyboards/dreamystate/dy16/info.json [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+          {"x":2, "y":3},
+          {"x":3, "y":3},
+		  {"x":0, "y":4},
+          {"x":1, "y":4},
+          {"x":2, "y":4},
+          {"x":3, "y":4}
+        ]
+      }
+    }
+  }
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/info.json:33]:
```suggestion
        "LAYOUT_ortho_5x4": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 1, "y": 0},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0},

                {"x": 0, "y": 1},
                {"x": 1, "y": 1},
                {"x": 2, "y": 1},
                {"x": 3, "y": 1},

                {"x": 0, "y": 2},
                {"x": 1, "y": 2},
                {"x": 2, "y": 2},
                {"x": 3, "y": 2},

                {"x": 0, "y": 3},
                {"x": 1, "y": 3},
                {"x": 2, "y": 3},
                {"x": 3, "y": 3},

                {"x": 0, "y": 4},
                {"x": 1, "y": 4},
                {"x": 2, "y": 4},
                {"x": 3, "y": 4}
            ]
        }
    }
}
```

File: keyboards/dreamystate/dy16/keymaps/default/keymap.c [hunk 15]
```diff
@@ -0,0 +1,28 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [0] = LAYOUT(
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/keymaps/default/keymap.c:20]:
```suggestion
  [0] = LAYOUT_ortho_5x4(
```

File: keyboards/dreamystate/dy16/keymaps/via/keymap.c [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+  ),
+  [2] = LAYOUT(
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS
+  ),
+
+  [3] = LAYOUT(
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/keymaps/via/keymap.c:42]:
```suggestion
  [0] = LAYOUT_ortho_5x4(
    KC_A, KC_B,  KC_C,  KC_D,
    KC_E, KC_F,  KC_G,  KC_H,
    KC_I, KC_J,  KC_K,  KC_L,
    KC_M, KC_N,  KC_O,  KC_P,
    KC_Q, KC_R,  KC_S,  KC_T
  ),
  [1] = LAYOUT_ortho_5x4(
    KC_TRNS, KC_TRNS  ,  RESET,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS
  ),
  [2] = LAYOUT_ortho_5x4(
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS
  ),

  [3] = LAYOUT_ortho_5x4(
```

File: keyboards/dreamystate/dy16/dy16.h [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( \
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/dy16.h:28]:
```suggestion
#define LAYOUT_ortho_5x4( \
```

File: keyboards/dreamystate/dy16/rules.mk [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/rules.mk:22]:
```suggestion
AUDIO_ENABLE = no           # Audio output

LAYOUTS = ortho_5x4
```

File: keyboards/dreamystate/dy16/readme.md [hunk 19]
```diff
@@ -0,0 +1,19 @@
+# Dreamystate Dy16 Keyboard
+
+![Dy16Photo](https://i.imgur.com/jpnMay9.jpeg)
```
Comment (noroadsleft) [inline keyboards/dreamystate/dy16/readme.md:3]: Swaps this image for a smaller version.

```suggestion
![Dy16Photo](https://i.imgur.com/jpnMay9m.jpeg)``
```

File: keyboards/dreamystate/dy16/readme.md [hunk 20]
```diff
@@ -0,0 +1,19 @@
+# Dreamystate Dy16 Keyboard
+
+![Dy16Photo](https://i.imgur.com/jpnMay9m.jpeg)``
```
Comment (noroadsleft) [inline keyboards/dreamystate/dy16/readme.md:3]: I had a typo on my previous suggestion.

```suggestion
![Dy16Photo](https://i.imgur.com/jpnMay9m.jpeg)
```

File: keyboards/dreamystate/dy16/readme.md [hunk 21]
```diff
@@ -0,0 +1,19 @@
+# Dreamystate Dy16 Keyboard
+
+![Dy16Photo](https://i.imgur.com/jpnMay9m.jpeg)
+More: [Imgur](https://imgur.com/gallery/6tu9KdA).
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/readme.md:4]:
```suggestion

More: [Imgur](https://imgur.com/gallery/6tu9KdA).
```

File: keyboards/dreamystate/dy16/readme.md [hunk 22]
```diff
@@ -0,0 +1,19 @@
...
+
+A 16 key macropad, with USB C, RGB light and Buzzer、 Encoder in the future.
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/readme.md:6]:
```suggestion
A 16 key macropad, with USB C, RGB light and Buzzer. Encoder in the future.
```

File: keyboards/dreamystate/dy16/dy16.h [hunk 23]
```diff
@@ -0,0 +1,42 @@
...
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_ortho_5x4( \
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/dy16.h:28]: The image in the readme shows a 4x4 layout - where's this extra row coming from?

Comment (DreamyState) [reply keyboards/dreamystate/dy16/dy16.h:28]: In 'qmk_firmware\keyboards\tetris', the author used the encoder to correspond to the row5 keys. I'm just making space for future upgrades.

File: keyboards/dreamystate/dy16/config.h [hunk 24]
```diff
@@ -0,0 +1,185 @@
...
+#define UNUSED_PINS
+
+/* Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+*/
```
Comment (fauxpark) [inline keyboards/dreamystate/dy16/config.h:43]:
```suggestion
/* Keyboard Matrix Assignments
 *
 * Change this to how you wired your keyboard
 * COLS: AVR pins used for columns, left to right
 * ROWS: AVR pins used for rows, top to bottom
 * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
 *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
*/
#define MATRIX_ROW_PINS { B7, D0, D1, D5, C7 }
#define MATRIX_COL_PINS { F7, F6, F5, F4 }
#define UNUSED_PINS
```

File: keyboards/dreamystate/dy16/dy16.h (hunk 23 above)
Comment (fauxpark) [reply keyboards/dreamystate/dy16/dy16.h:28]: This is what revision folders are for. If you're planning multiple revisions, you should make this board 4x4, and then place the configuration specific to this revision under `rev1`, etc.

Comment (DreamyState) [reply keyboards/dreamystate/dy16/dy16.h:28]: I see. When my code is finished, I will upload it directly. thank you.

File: keyboards/dreamystate/dy16/config.h [hunk 25]
```diff
@@ -0,0 +1,210 @@
...
+#define ENCODER_RESOLUTION 4
+
+/* define if matrix has ghost (lacks anti-ghosting diodes) */
+//#define MATRIX_HAS_GHOST
+
+/* number of backlight levels */
+// #define BACKLIGHT_PIN B6
+// #define BACKLIGHT_BREATHING
+// #define BACKLIGHT_LEVELS 5
+
```
Comment (noroadsleft) [inline keyboards/dreamystate/dy16/config.h:109]: Don't need this as the board doesn't support Backlight (single-color non-per-key).

```suggestion
```

File: keyboards/dreamystate/dy16/config.h [hunk 26]
```diff
@@ -0,0 +1,210 @@
...
+//#define MAGIC_KEY_LAYER8         8
+//#define MAGIC_KEY_LAYER9         9
+//#define MAGIC_KEY_BOOTLOADER     B
+//#define MAGIC_KEY_BOOTLOADER_ALT ESC
+//#define MAGIC_KEY_LOCK           CAPS
+//#define MAGIC_KEY_EEPROM         E
+//#define MAGIC_KEY_EEPROM_CLEAR   BSPACE
+//#define MAGIC_KEY_NKRO           N
+//#define MAGIC_KEY_SLEEP_LED      Z
+
```
Comment (noroadsleft) [inline keyboards/dreamystate/dy16/config.h:190]:
```suggestion
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/dreamystate/dy16/keymaps/via/keymap.c [hunk 27]
```diff
@@ -0,0 +1,78 @@
...
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS
+  )
+};
+
+void encoder_update_user(uint8_t index, bool clockwise) {
+  uint8_t layer = biton32(layer_state);
```
Comment (drashna) [inline keyboards/dreamystate/dy16/keymaps/via/keymap.c:52]:
```suggestion
  uint8_t layer = get_highest_layer(layer_state);
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**